lib/netext/httpext/transport.go

```diff
@@ -80,75 +80,51 @@ func (t *transport) measureAndEmitMetrics(unfReq *unfinishedRequest) *finishedRe
 		trail:             trail,
 	}
 
-	tags := t.tags
+	ctm := t.tagsAndMeta.Clone()
```

Comment:
I believe the context of the usage of this variable is big enough to give it a full name like `clonedTagsAndMetadata` (or just a `clonedTags`) or even better to call it `sampleTags` (which I guess it is) since it can be essential to know downstream.

Reply:
Moved into #2726